app/build.gradle

```diff
@@ -42,7 +42,7 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
         debug {
-            minifyEnabled true
+            minifyEnabled false
```

Comment:
This was breaking the instrumentation tests and it wasn't quick to figure out how to fix proguard.